modules/tinymce/src/themes/silver/main/ts/ContextToolbar.ts

```diff
@@ -310,7 +326,7 @@ const register = (editor: Editor, registryContextToolbars, sink: AlloyComponent,
     editor.on('focusout', (_e) => {
       Delay.setEditorTimeout(editor, () => {
         if (Focus.search(sink.element).isNone() && Focus.search(contextbar.element).isNone()) {
-          close();
+          // close();
```

Comment:
Reinstate this. You just comment this out when you want a less painful debugging experience.